substrate/utils/frame/benchmarking-cli/src/storage/cmd.rs

```diff
@@ -116,6 +117,21 @@ pub struct StorageParams {
 	/// Include child trees in benchmark.
 	#[arg(long)]
 	pub include_child_trees: bool,
+
+	/// Enable PoV recorder.
+	///
+	/// The recorder has impact on performance when benchmarking with the TrieCache enabled.
+	/// For parachains weights this should be true and false for relay chain weights.
+	#[arg(long, default_value = "true", action=ArgAction::Set)]
```

Comment:
```suggestion
	#[arg(long, default_value_t = true)]
```

Reply:
Maybe this should be called `disable-pov-recorder` and then the recorder is deactivated.